Firstcomponent.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+	<link rel="stylesheet" type="text/css" href="first.css">
+	<title></title>
+</head>
+<body>
+
+<div id="floating-subscribe" class="floatingSubscribeVisible floatingTransition  hide">
+	<div class="floating-subscribe-wrapper">
+		<div class="floating-subscribe-close" onclick="myFunction()">
+			<img src="img1.svg" alt="Close">
```

Comment:
Ippa github la upload pannitom la - intha image ku oru url vanthirukum da. Shorten that ur using http://bit.ly/ (give some meaningful name) and use for images

Move all images to new folder assets